src/pages/personal-settings/index.js

```diff
@@ -0,0 +1,154 @@
+import { useEffect, useState, useContext } from 'react'
+import { Grid, Box } from '@mui/material'
+import { useFormik } from 'formik'
+import toast from 'react-hot-toast'
+import ErrorWindow from 'src/components/Shared/ErrorWindow'
+import WindowToolbar from 'src/components/Shared/WindowToolbar'
+import CustomTextField from 'src/components/Inputs/CustomTextField'
+import { RequestsContext } from 'src/providers/RequestsContext'
+import { SelfServiceRepository } from 'src/repositories/SelfServiceRepository'
+import { ResourceIds } from 'src/resources/ResourceIds'
+import ResourceComboBox from 'src/components/Shared/ResourceComboBox'
+import { useResourceQuery } from 'src/hooks/resource'
+import { DataSets } from 'src/resources/DataSets'
+import * as yup from 'yup'
+
+const PersonalSettings = () => {
+  const [errorMessage, setErrorMessage] = useState(null)
+  const { getRequest, postRequest } = useContext(RequestsContext)
+
+  const [initialValues, setInitialValues] = useState({
+    recordId: null,
+    email: '',
+    activeStatus: '',
+    userType: '',
+    homePage: '',
+    username: '',
+    fullName: '',
+    languageId: '',
+    menuTheme: '',
+    languageName: ''
+  })
+
+  useEffect(() => {
+    ;(async function () {
+      try {
+          const userData = window.sessionStorage.getItem('userData')
+          ? JSON.parse(window.sessionStorage.getItem('userData'))
+          : null;
+          const _userId = userData && userData.userId;
+
+          const res = await getRequest({
+            extension: SelfServiceRepository.SSUserInfo.get,
+            parameters: `_recordId=${_userId}`
+          })
+
+          setInitialValues(res.record)
+        
+      } catch (exception) {}
+    })()
+  }, [])
+
+  const { labels: _labels } = useResourceQuery({
+    datasetId: ResourceIds.PersonalSettings
+  })
+
+  const formik = useFormik({
+    enableReinitialize: true,
+    validateOnChange: true,
+    initialValues,
+    validationSchema: yup.object({
+      fullName: yup.string().required(' '),
+      languageId: yup.string().required(' ')
+    }),
+    onSubmit: values => {
+        postPersonalSettings(values)
+    }
+  })
+  
+  const postPersonalSettings = obj => {
+   
+    postRequest({
+      extension: SelfServiceRepository.SSUserInfo.set,
+      record: JSON.stringify(obj)
+    })
+      .then(res => {
+        if (res) toast.success('Record Added Successfully')
+      })
+  }
+
+  const handleSubmit = () => {
+    formik.handleSubmit()
+  }
+
+  return (
+    <>
+      <Box
+        sx={{
+          display: 'flex',
```

Comment:
Do we have MUI specific attributes that we can replace those with?

Reply:
i changed both in box and grid container at the bottom of page